pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -170,6 +170,14 @@ func (alloc *Action) Execute(ssn *framework.Session) {
 		}
 
 		job := jobs.Pop().(*api.JobInfo)
+		if ssn.OverusedAfterJobEnqueue(job) {
+			jobs.Push(job)
+			klog.V(3).Infof("Queue <%s> is overused when considering job <%s>, ignore it.", queue.Name, job.Name)
+			delete(queueInNamespace, queue.UID)
+			namespaces.Push(namespace)
+			continue
```

Comment:
Don't delete queue from queueInNamespace bacause there are other jobs in queue;
the job in status overuse shouldn't be re-pushed.